main.nf

```diff
@@ -571,6 +572,8 @@ if(params.aligner == 'star'){
             --readFilesCommand zcat \\
             --runDirPerm All_RWX \\
             --outFileNamePrefix $prefix $seqCenter \\
+            
+        samtools index ${prefix}Aligned.sortedByCoord.out.bam \\
```

Comment:
Instead of the whitespace, better to just remove the double backslashes..
```suggestion
        samtools index ${prefix}Aligned.sortedByCoord.out.bam
```